server/spec/content_access_spec.rb

```diff
@@ -558,4 +558,94 @@ def read_json_file(filename)
     expect(content['vendor']).to eq(@content.vendor)
     expect(content['path']).to eq(@content.contentUrl)
   end
+
+  it 'should honour the content defaults for owner in SCA mode' do
+    product = create_product('test-product-p1', 'some product-p1')
+
+    # Content enabled = true
+    content_c1 = @cp.create_content(
+        @owner['key'], "cname-c1", 'test-content-c1', random_string("clabel"), "ctype", "cvendor",
+        {:content_url=> '/this/is/the/path',  :modified_products => [@modified_product["id"]]}, true)
+    @cp.add_content_to_product(@owner['key'], product['id'], content_c1['id'], true)
+
+    # Content enabled = false
+    content_c2 = @cp.create_content(
+        @owner['key'], "cname-c2", 'test-content-c2', random_string("clabel"), "ctype", "cvendor",
+        {:content_url=> '/this/is/the/path',  :modified_products => [@modified_product["id"]]}, true)
+    @cp.add_content_to_product(@owner['key'], product['id'], content_c2['id'], false)
+
+    @consumer = consumer_client(@user, @consumername, type=:system, username=nil, facts= {'system.certificate_version' => '3.3'})
+    certs = @consumer.list_certificates
+    certs.length.should == 1
+    cert = certs[0]['cert']
+    json_body = extract_payload(cert)
+    # Check content status
+    json_body['products'][0]['content'].each do |content|
+      if content.id == content_c1.id
+        expect(content.enabled).to be_nil
```

Comment:
This is weird. If a content is enabled, we should expect true here, not false, but it doesn't look like this is caused by your changes. I'll look more into this